app/helpers/movie_person_profiles_helper.rb

```diff
@@ -7,4 +7,8 @@ def display_birthday_info(profile)
     date = profile.birthday.to_date.stamp('January 1st, 2018')
     profile.deathday.blank? ? "#{date} (Age: #{profile.age})" : "#{date}, Deceased (Age: #{profile.age})"
   end
+
+  def actor_movie_posters_uri(actor)
```

Comment:
Was reading the [difference between](https://stackoverflow.com/questions/176264/what-is-the-difference-between-a-uri-a-url-and-a-urn) `uri` and `url` and 🤷 . I don't care, and you don't need to explain it here, but just curious.

Reply:
Though it bothered me, I made that decision for 3 reasons:
* I didn't want any confusion between that helper method and a rails-generated  path helper -- for the sake of troubleshooting.
* I knew it wasn't _wrong_ even though it didn't feel like the right name overall
* naming is extra hard when you're extra tired 😆 😪